packaging/src/docker/6.x/.env

```diff
@@ -1,6 +1,5 @@
-# In order to start the community edition set the value of ALFRESCO_EDITION to -community
-ALFRESCO_EDITION=
```

Comment:
Can the ALFRESCO_EDITION parameter be put back please?

Reply:
From ACS 6.1 Docker Compose is not valid, as it requires Transform Services to work (Tika, LibreOffice...). Is this required for some build in Bamboo?